src/xtgeo/common/sys.py

```diff
@@ -159,9 +159,11 @@ def __init__(self, fobj, mode="rb", obj=None):
         elif isinstance(fobj, io.BytesIO):
             self._file = fobj
             self._memstream = True
+            self._file.seek(0)
```

Comment:
This creates the same problem that the user is unable to point to the start of the content in the stream. Position 0 might not be the start of the content xtgeo should read

Reply:
OK